src/test/java/seedu/address/logic/parser/ExportCommandParserTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.logic.commands.ExportCommand.MESSAGE_USAGE;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.ExportCommand;
+
+/**
+ * Test cases for export command.
+ */
+public class ExportCommandParserTest {
```

Comment:
Should `export email phone` and `export phone email` be parsed successfully?

Reply:
Nope, a parse exception will be thrown in ExportCommand#execute()

Reply:
I added the comment for handling invalid command format in `ExportCommandParser`. 
Perhaps can consider adding test case for it.

Reply:
Oh I mean the PR review comment for `ExportCommandParser`. 
"Should `parse` method handle exception related to the invalid command format? As ExportCommandParser parses the command."
If you throw `ParseException` in `ExportCommandParser`, then perhaps can add `parse_invalidArgs` test case?

Reply:
Oh nope, ExportCommandParse now does not throw any exceptions. It leaves the validation to ExportCommand